examples/NonArduino/Tock/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.18)
+
+# create the project
+project(tock-sx1261)
+
+set(LINKER_SCRIPT ${CMAKE_CURRENT_SOURCE_DIR}/libtock-c/userland_generic.ld)
+
+include("tock.cmake")
+
+# when using debuggers such as gdb, the following line can be used
+#set(CMAKE_BUILD_TYPE Debug)
+
+# if you did not build RadioLib as shared library (see wiki),
+# you will have to add it as source directory
+# the following is just an example, yours will likely be different
+add_subdirectory("${CMAKE_CURRENT_SOURCE_DIR}/../../../../RadioLib" "${CMAKE_CURRENT_BINARY_DIR}/RadioLib")
+
+# add the executable
+add_executable(${PROJECT_NAME} main.cpp)
+
+# link with RadioLib and libtock-c
+target_link_libraries(${PROJECT_NAME} PUBLIC
+    RadioLib
+    gcc
```

Comment:
Is linking to gcc really needed? Just curious.

Reply:
Yeah it is. libtock-c requires some of the helper functions provided